src/Validator/Rules/ValuesOfCorrectType.php

```diff
@@ -253,24 +271,24 @@ static function (EnumValueDefinition $value) : string {
                 )
             );
 
-            return $suggestions ? 'Did you mean the enum value ' . Utils::orList($suggestions) . '?' : null;
+            return $suggestions ? static::$enumTypeSuggestionMessage . Utils::orList($suggestions) . '?' : null;
```

Comment:
I think this one should be replaced with `sprintf`, grammar may not be the same in all languages.

Reply:
Replaced.